src/Eleventy.js

```diff
@@ -759,7 +780,18 @@ Arguments:
     await this.config.events.emit("beforeBuild");
 
     try {
-      let promise = this.writer.write();
+      let promise;
+      if (to === "fs") {
+        promise = this.writer.write();
+      } else if (to === "json") {
+        promise = this.writer.getJSON("json");
+      } else if (to === "ndjson") {
+        promise = this.writer.getJSON("ndjson");
+      } else {
+        throw new Error(
+          `Invalid argument for \`Eleventy->executeBuild(${to})\`, expected "json" or "fs".`
```

Comment:
Q: Does this error msg need to add the new "ndjson" option?

Reply:
Yes thank you!